apps/prairielearn/src/schemas/schemas/infoQuestion.json

```diff
@@ -115,7 +117,10 @@
         },
         "environment": {
           "description": "Environment variables to set inside the grading container.",
-          "type": "object"
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
```

Comment:
This and the corresponding change below technically have the potential to be breaking if someone had done something like `{"foo": true, "bar": 1}`. However, I used the schema checking tooling introduced in #11024 and didn't see any impact in practice.